### PR TITLE
Add changelog and the updated pull request template

### PR DESCRIPTION
**Pull request by aashikam**
## Purpose
$title

## Remarks:
https://github.com/ballerina-platform/ballerina-standard-library/issues/1198
https://github.com/ballerina-platform/ballerina-standard-library/issues/1197

### CODE REVIEW

**Comment by ThisaruGuruge** [comment]
Shall we delete the existing pull_request.md and issue_template.md files with this?

File: changelog.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Change Log
+This file contains all the notable changes done to the Ballerina NATS package through the releases.
+
+## 1.1.0-alpha6
```
Comment (ThisaruGuruge) [inline changelog.md:4]: Shall we add the date here?

Comment (aashikam) [reply changelog.md]: Added 67f1e6cfc3a38cedc5fd9c30a302effb8c6ee327

**Review by ThisaruGuruge — APPROVED** [review]
LGTM

**Pull request merged**